### PR TITLE
[README.md] Disabling bundles

### PR DESCRIPTION
**Pull request by begin-theadventure**

### CODE REVIEW

**Comment by barnabwhy** [comment]
You can instead just do `yarn tauri build --bundles none` or `yarn tauri build -b none`

File: README.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ You can also download the [latest release](https://github.com/harmonytf/HarmonyV
 1. Run `yarn`
 3. Run `yarn tauri build` to build a portable executable and bundled installs for your OS
 
+To build only the executable, change `bundle` `active` to `false` in `src-tauri/tauri.conf.json`.
```
Comment (barnabwhy) [inline README.md]:
```suggestion
To build only the executable, instead run `yarn tauri build -b none`.
```

File: README.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ You can also download the [latest release](https://github.com/harmonytf/HarmonyV
...
+
+On Linux, it can be done with: `sed -i '/"bundle": {/,/},/{/"active":/s/true/false/} src-tauri/tauri.conf.json`
```
Comment (barnabwhy) [inline README.md]:
```suggestion
```

**Review by barnabwhy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (begin-theadventure) [reply README.md]: COOL

**Pull request merged**